docs/guidelines/platforms/android/guideline_understandable_android.md

```diff
@@ -2,13 +2,15 @@
 
 # Understandable guidelines for Android
 
-## Readable
+## Readable (WCAG 3.1)
 
 Make text content readable and understandable.
 
-### Language of the App
+### Language of Page (WCAG 3.1.1 -Level A)
 
-*This guideline covers point 3.1.1 Language of Page - Level A of the WCAG standard.*
+This guideline covers point [3.1.1 Language of Page - Level A](https://www.w3.org/WAI/WCAG22/quickref/#language-of-page) of the WCAG standard.
+
+The default human language of an App can be programmatically determined.
 
 :white_check_mark: **Success criteria**
```

Comment:
Does it make sense to you that we somehow include our library [Localian](https://github.com/infinum/android-localian) in this section, as some kind of helper for handling locales?

Reply:
I believe we could mention it as an example of  library that can simplify this logic 👍